### DISCUSSION TITLE
Behaviour with three displays

### DISCUSSION THREAD

**Discussion by Milliw** [Noobsai/fullscreen-avoider#20] [General]
I do have three displays and since a clean install of my distro the bar moves to the "wrong" display.
How can I alter this? 

I did it once but can't figure it out anymore

**Comment by thevoidside** [comment]
I have also had some interesting behavior with this extension with 3 monitors. When I first installed it, it would move the top bar to Display 2. Then, out of nowhere, it started going instead to Display 3. Now it's back on Display 2 again... It really would be nice if there was a toggle in the extension options to choose which monitor you wanted the top bar to move to.
